CollAction/Services/Project/ProjectService.cs

```diff
@@ -83,38 +83,42 @@ public async Task<ProjectParticipant> GetParticipant(string userId, int projectI
             return await _context.ProjectParticipants.SingleOrDefaultAsync(p => p.ProjectId == projectId && p.UserId == userId);
         }
 
-        public async Task<IEnumerable<FindProjectsViewModel>> FindProjects(Expression<Func<Models.Project, bool>> filter)
+        public IQueryable<FindProjectsViewModel> FindProjects(Expression<Func<Models.Project, bool>> filter, int? limit)
         {
-            return await _context.Projects
+            IQueryable<Models.Project> projects = _context.Projects
```

Comment:
Is it possible for us to import the CollAction.Models namespace so that we can omit it from here?

Reply:
There's also a namespace "Collaction.Services.Project" which clashes with the model name. Which could rename one of them, but this solution is fine as well.

Reply:
+1